spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/transaction/ReactiveFirestoreTransactionManager.java

```diff
@@ -181,6 +186,7 @@ public boolean isRollbackOnly() {
 			return false;
 		}
 
+		//not supported in Firestore
```

Comment:
Change to Javadoc style comment, so it turns up in autogenerated docs.

Reply:
done